server/src/routes/api/v1/quotesRouter.js

```diff
@@ -0,0 +1,17 @@
+import express from "express";
+import got from "got";
+
+const quotesRouter = new express.Router();
+
+quotesRouter.get("/", async (req, res) => {
+  try {
+    const url = 'https://api.quotable.io/random'
```

Comment:
as a good exercise, extract out an ApiClient for retrieving this quote, and consider writing a unit test that does not hit the API endpoint itself.